src/doc/book/crates-and-modules.md

```diff
@@ -280,6 +290,20 @@ functionality is in a library crate, and the executable crate uses that
 library. This way, other programs can also use the library crate, and it’s also
 a nice separation of concerns.
 
+Since we have added a `main.rs` file, we need to change the `Cargo.toml` a bit, like this:
```

Comment:
@Manishearth: It is, but from my point of view it is more comfortable to read as I proposed.